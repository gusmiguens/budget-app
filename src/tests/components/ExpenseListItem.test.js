const React = require('react')
const { shallow } = require('enzyme')
const ExpenseListItem = require('../../components/ExpenseListItem')
const expenses = require('../fixtures/expenses')

describe('<ExpenseListItem />', () => {
    it('Should render ExpenseListItem correctly', () => {
        const wrapper = shallow(<ExpenseListItem {...expenses[1]} />)
        expect(wrapper).toMatchSnapshot()
    })

})