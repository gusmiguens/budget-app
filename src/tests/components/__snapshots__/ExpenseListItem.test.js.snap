// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ExpenseListItem /> Should render ExpenseListItem correctly 1`] = `
<div>
  <Link
    to="/edit/2"
  >
    <h3>
      Rent
    </h3>
  </Link>
  <p>
    Amount: 
    2000
     - CreatedAt: 
    -345600000
  </p>
</div>
`;
